core/src/test/scala/org/apache/spark/ContextCleanerSuite.scala

```diff
@@ -245,7 +245,7 @@ class ContextCleanerSuite extends ContextCleanerSuiteBase {
     assert(fs.exists(RDDCheckpointData.rddCheckpointDataPath(sc, rddId).get))
 
     // Test that GC causes checkpoint data cleanup after dereferencing the RDD
-    postGCTester = new CleanerTester(sc, Seq(rddId), Nil, Nil)
+    postGCTester = new CleanerTester(sc, Seq(rddId), Nil, Nil, Seq(rddId))
```

Comment:
The tips are more confusing. 
The [jira](https://issues.apache.org/jira/browse/SPARK-6963) display the error in: [ContextCleanerSuite.scala#L252](https://github.com/witgo/spark/blob/SPARK-6963/core/src/test/scala/org/apache/spark/ContextCleanerSuite.scala#L252).